system-variables.md

```diff
@@ -772,6 +772,14 @@ Constraint checking is always performed in place for pessimistic transactions (d
 >
 > Only the default value of `OFF` can be considered safe. Setting `tidb_enable_noop_functions=1` might lead to unexpected behaviors in your application, because it permits TiDB to ignore certain syntax without providing an error. For example, the syntax `START TRANSACTION READ ONLY` is permitted, but the transaction remains in read-write mode.
 
+### tidb_enable_paging <span class="version-mark">New in v5.4.0</span>
+
+- Scope: SESSION | GLOBAL
+- Default value: `OFF`
+- This variable controls whether to use paging to send coprocessor requests in `IndexLookUp` operator.
```

Comment:
"whether to use the method of paging to send coprocessor"
Is this better?

Reply:
Thanks. Fixed.